packages/slice-machine/src/legacy/lib/models/common/widgets/Group/ListItem/index.jsx

```diff
@@ -153,6 +153,13 @@ export const CustomListItem = ({
       apiId: groupItem.key,
       newKey: groupItem.key,
       value: Groups.toSM(newGroupValue),
+      groupId: groupItem.key,
+      updateMeta: {
+        fieldDeletedOrIdChanged: {
+          previousPath: [groupItem.key, key],
+          newPath: null,
+        },
+      },
```

Comment:
"update meta" sounds like a verb/action

i'd call it just `updates`, to be declarative

and i'd also use an object with key/value, key being the current path and value the new one

it works for deletes too, by just using null as a value

```suggestion
      updates: {
        [[groupItem.key, key].join("#")]: null
      },
```

Reply:
1. why would you need a null as a current path? i don't think we should complicate this more than the strictly necessary today
2. for the same reasons as above, if you already can think of a scenario that we'll need to address within 3 months, i'm happy to give it a thought, but otherwise i'd use the simpler name and data structure possible today, and change it in the future if needed
3. updateDetails is also generic, misleading (what are "details" anyway?) and again looks like an action, not a declarative property as we need

i think `updates` with the structure i shared above (and without `fieldDeletedOrIdChanged`) feels the nicest to me, but feel free to keep the current one or anything else, i'll let you folks decide

Reply:
I'm not against it, I just had a different idea for the property, something like an update context bag, but I will try your approach.